app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
```

Comment:
- [ ] The `likes` belong to the `author` and post, kindly make sure to set up associations for this so it works properly